winui/Avatar-view/content-types.md

```diff
@@ -0,0 +1,235 @@
+---
+layout: post 
+title: Content Types in WinUI AvatarView control | Syncfusion
+description: Learn about the available content types and how to set values for them in the WinUI AvatarView control.
+platform: WinUI
+control: AvatarView
+documentation: ug
+---
+
+# Content Types in WinUI AvatarView
+
+The `AvatarView` control supports five different types of content: `Default`, `Initials`, `CustomImage`, `AvatarCharacter`, and `GroupView`.
```

Comment:
The `AvatarView` control supports five types of content: `Default,` `Initials,` `CustomImage,` `AvatarCharacter,` and `GroupView.`